src/transformers/cache_utils.py

```diff
@@ -536,6 +538,57 @@ def batch_select_indices(self, indices: torch.Tensor):
             self.value_cache[layer_idx] = self.value_cache[layer_idx][indices, ...]
 
 
+if is_torch_greater_or_equal_than_2_6:
```

Comment:
can you add a comment why this requires 2.6

Reply:
DOne